src/lib.rs

```diff
@@ -62,6 +62,10 @@
 //! by enabling the `dummy` feature, which will make `getrandom` to use an
 //! always failing implementation.
 //!
+//! If you are seeing this compiler error even though your crate doesn't directly
+//! depend on getrandom (one of your dependencies pulls it in), you can still solve
+//! this by adding getrandom as a dependency to your `Cargo.toml` with the correct feature.
```

Comment:
I don't think there is **the** feature to be enabled. Also please take a look at #87. I am not sure if enabling `dummy`, `stdweb` or `wasm-bindgen` feature in an non-application `Cargo.toml` is a correct recommendation.